package.json

```diff
@@ -5,9 +5,9 @@
   "main": "./lib/index.js",
   "scripts": {
     "lint": "eslint .",
-    "prepublish": "babel --out-dir ./lib ./src",
+    "prepublish": "NODE_ENV=development babel --out-dir ./lib ./src",
```

Comment:
@SamyPesse this is kinda where I got lost in how your library works when it's npm installed etc. Can you highlight how the `./lib` folder serves files considering they don't exist?